src/index.html

```diff
@@ -15,8 +15,54 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="Moyo-logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="is-active section">Apple</a>
```

Comment:
The anchor tags for navigation items should have 'href' attributes to be functional links. Consider adding appropriate URLs or placeholders like '#'.